scipy-1.0/scipy-optimize.tex

```diff
@@ -130,3 +130,10 @@ \subsubsection*{Nonlinear Optimization}
   \end{tabular}
   \label{tab:minimize}
 \end{table}
+
+Some problems may require the use of global optimizers to find the minimum of the objective function over the set of all parameter values, especially if the objective function possesses many local minima that other solvers could get stuck in.
```

Comment:
I think it is worth mentioned that this global solvers are made available apart from the ``minimize`` interface.
Since we were talking about the minimize interface before, this may generate some confusion, if not explicitly mentioned.